### DISCUSSION TITLE
Why not a permissive license?

### DISCUSSION THREAD

**Discussion by TheButlah** [brndnmtthws/dryoc#61] [General]
Hi, I was considering this for use at work but we can only consume permissively licensed code. libsodium for example is ISC licensed which is similarly permissive to MIT. But dryoc is LGPL, which would require our  code to *also* be lgpl if we statically link (which we want to do for distribution reasons), even if we don't modify the original source code. Most of our code is MIT/Apache but we have a few places where we use closed source vendor SDKs that we don't control and don't have legal rights to open source. So using an LGPL library would be impossible for us.

Would the author be willing to consider licensing the same as the rest of the rust ecosystem, I.e. a dual Apache 2.0/MIT license? These would be most preferable because they don't impose any restrictions on users of the code.

**Comment by brndnmtthws** [comment]
Here you go because you asked so nicely: https://github.com/brndnmtthws/dryoc/pull/62

**Reply by TheButlah** [reply]
So awesome. Thank you ❤️
